test/e2e/helpers.js

```diff
@@ -724,7 +724,10 @@ async function clickSignOnSignatureConfirmation({
     // so we have to add a small delay as the last alternative to avoid flakiness.
     await driver.delay(regularDelayMs);
   }
-
+  await driver.waitForSelector(
+    { text: 'Sign', tag: 'button' },
+    { state: 'enabled' },
```

Comment:
Minor, can we add this argument to `clickElement` to avoid the duplication?

Assuming `clickElement` isn't checking the enabled state.